src/components/List.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+class List extends React.Component {
+
+  render () {
+    const {
+      tabs,
+      onTabSelected,
+    } = this.props;
+    return (
+      <ul className="Tab__list List">
+        {tabs.map((tab) => (
+            <li
+              className={`List__item List__item--${tab.index + 1}`}
+              key={tab.index}
+              onClick={
+                onTabSelected
+              }
```

Comment:
```suggestion
              onClick={() => onTabSelected(tab.index)}
```
You can pass selected tab index here. No need to search for it in the App component